src/main/resources/META-INF/resources/primefaces/photocam/photocam.js

```diff
@@ -1092,12 +1103,19 @@ PrimeFaces.widget.PhotoCam = PrimeFaces.widget.BaseWidget.extend({
         if (!("autoStart" in this.cfg)) {
             this.cfg.autoStart = true;
         }
+        
```

Comment:
Could you add the new properties of the widget configuration to the JavaScript docs around line 1073? Like this:

```
 * @prop {string} cfg.device Suggests a video input device such as `user` (font camera) or `environment` (rear camera).
 * @prop {string} cfg.onCameraError Client side callback executed if the camera has an error. This string must be valid
 * JavaScript code. It is executed within the context of a function with a single parameter `errorObj` of type `Error`.
```